next-tavla/app/(admin)/boards/page.tsx

```diff
@@ -20,13 +20,19 @@ async function OrganizationsBoardsPage() {
     if (!user) redirect('/')
 
     const boardsWithOrg = await getAllBoardsForUser()
-
+    const tags = [
+        ...new Set(
+            boardsWithOrg
+                .map((boardWithOrg) => boardWithOrg.board.meta?.tags ?? [])
+                .flat(),
```

Comment:
Kan benytte `flatMap() `her i stedet for `.map()` og `flat()`. Jeg synes det er ryddigst å bruke `uniq()`-metoden til lodash også - litt lettere å lese

```
    const uniqueTags = uniq(
        boardsWithOrg.flatMap(
            (boardWithOrg) => boardWithOrg.board?.meta?.tags ?? [],
        ),
    )
```